client/src/VideoComponent.js

```diff
@@ -0,0 +1,155 @@
+// version5 this works on all. but not same size youtube videos
+import React from "react";
+import axios from "axios";
+import "./App.css";
+
+const VideoComponent = ({ video, onUpVote, onDownVote, onRemove }) => {
+  const handleDelete = async () => {
+    try {
+      await axios.delete(
+        `https://back-end-full-stack-project-assessment.onrender.com/videos/${video.id}`
+      );
+      onRemove(video.id);
+    } catch (error) {
+      console.error("Error deleting video:", error);
+    }
+  };
+
+  return (
+    <div className="video-component">
+      <h3>{video.title}</h3>
+      <iframe
+        title={video.title}
+        width="560"
+        height="315"
+        src={`https://www.youtube.com/embed/${video.url.split("v=")[1]}`}
```

Comment:
What would happen if the video URL didn't include `v=`? Is there anything we could do to make that better?